pycrostates/viz/tests/test_cluster_centers.py

```diff
@@ -1,14 +1,20 @@
+import logging
+
 import numpy as np
 import pytest
 from matplotlib import pyplot as plt
 from mne import create_info
 
 from pycrostates.io import ChInfo
+from pycrostates.utils._logs import logger
```

Comment:
```suggestion
from pycrostates import logger
```

Defined at the top-level of the package, right?